recipes/footprint/meta.yaml

```diff
@@ -8,6 +8,7 @@ source:
 
 build:
   skip: True  # [not linux]
+  noarch: generic
```

Comment:
noarch? what a magic

Reply:
the wraps different tools and R packages and only tested on Linux. noarch is not true here.